Assets/HoloToolkit/Utilities/Scripts/Editor/AutoConfigureWindow.cs

```diff
@@ -123,12 +147,24 @@ protected virtual void OnGUI()
             GUILayout.EndScrollView();
             GUILayout.EndVertical();
 
-            // Apply button
-            GUILayout.BeginVertical(EditorStyles.miniButtonRight);
+            // Buttons
+            GUILayout.BeginHorizontal(EditorStyles.miniButtonRight);
+            bool selectAllClicked = GUILayout.Button("Select All");
+            bool selectNoneClicked = GUILayout.Button("Select None");
             bool applyClicked = GUILayout.Button("Apply");
-            GUILayout.EndVertical();
+            GUILayout.EndHorizontal();
```

Comment:
Why did this change?  Did we change the `StartVertical` to `StartHorizontal`?

Reply:
Yes, four lines above this one, at line 151. I changed it so the three buttons were aligned horizontally instead of stacked vertically.

Reply:
I'm blind 😆